### DISCUSSION TITLE
How do I set the local model to chat instead of text generation?

### DISCUSSION THREAD

**Discussion by tp1415926535** [Vali-98/ChatterUI#118] [Q&A] [answered]
The App Mode is ‘Local’, and selected the local .gguf model, the other configurations are default. But it just generates its own text during the dialogue.
<img src="https://github.com/user-attachments/assets/4abe7e0a-444e-4ef8-afae-ceeb862f4a70" alt="Screenshot_20241122_232702" width="300px">
How should I set it up?

**Comment by Vali-98** [comment, marked as answer]
At the moment, local generations only use text completion.

I would recommend looking at your model's stop tokens to properly stop said model from further generations.

**Reply by tp1415926535** [reply]
Maybe there is something wrong with the model I chose called `**-Instruct-GGUF`, it doesn't show the stop sequence either. I found a new one called `**-Chat-GGUF` and it seems to work fine.
